### DISCUSSION TITLE
How(where) to define own custom (global) function?

### DISCUSSION THREAD

**Discussion by gvanto** [pimcore/pimcore#13918] [Ideas]
For ease of development, I'd like to put a function dd($var) (as below) somewhere in pimcore (using 5.8) where it can be accessed globally...

```
function dd($var) {
   dump($var);die(0);
}
```

Any help much appreciated

**Comment by jdreesen** [comment]
You can define it in any (new) file you want and then [load that file via composer](https://getcomposer.org/doc/04-schema.md#files) (but I'd put it in [`autoload-dev`](https://getcomposer.org/doc/04-schema.md#autoload-dev), though).

**Reply by gvanto** [reply]
Cool thanks @jdreesen will give this a bash

**Comment by gvanto** [comment]
Strange how composer's default  "composer install" will INCLUDE dev requirements (including autoload-dev). To exclude them need to run with --no-dev which is weird would think this would be default ...
